projects/packages/sync/.gitignore

```diff
@@ -0,0 +1 @@
+wordpress
```

Comment:
Since you're adding a new `.gitignore` file, you'll need to add it to `.gitattributes`.

You can add `phpunit.xml.dist`, and  `tests/` in there too.

Reply:
👍